metadata-ingestion/src/datahub/ingestion/source/confluent_schema_registry.py

```diff
@@ -231,16 +234,24 @@ def get_schemas_from_confluent_ref_json(
         return all_schemas
 
     def _get_schema_and_fields(
-        self, topic: str, is_key_schema: bool
+        self, dataset_subtype: str, is_key_schema: bool, is_subject: bool
```

Comment:
Can we keep the topic/subject name here as well?
It is technically that, right?

Reply:
or maybe name it to `subject` or  `schema_name` ?

subtype I think is a bit misleading

Reply:
`is_subject` might be named as `is_kafka_topic` or `is_topic`, but I'm fine with the `is_subject` as well.
The name of the parameter should somehow describe it means if this is set we assume you are talking about a Kafka topic